quicksearch/solr/upgrade/ingest/solrconfig.xml

```diff
@@ -126,9 +126,17 @@
                 title_hl
                 author_hl
                 container_hl
-                subject_hl       
+                subject_name_hl
+                subject_topic_hl
+                date_hl
+                call_number_hl
+                access_hl
```

Comment:
access_hl was not mapped in dcs_mapper. Not sure if it is necessary because it is not in the MVP highlighted field list

Reply:
I believe MVP changed to highlight all fields even this one. I think we should add it now since I don't think there is a downside to adding it. I will create a PR to map it in the dcs mapper